src/models/User.js

```diff
@@ -159,8 +164,11 @@ class User extends Resource {
     return this.client.activateUser(this.id, queryParameters);
   }
 
-  deactivate() {
-    return this.client.deactivateUser(this.id);
+  /**
+   * @param {object} queryParameters
+   */
+  deactivate(queryParameters) {
+    return this.client.deactivateUser(this.id, queryParameters);
```

Comment:
same question, breaking changes?

Reply:
No issues with adding a parameter, they can just be omitted